docs/OV_Runtime_UG/Integrate_with_customer_application.md

```diff
@@ -66,235 +61,124 @@ Use the following code to create Inference Engine Core to manage available devic
 Optionally, configure input and output of the model using the steps below:
 
 1. Load a model to a Core object:
-   @sphinxdirective
-   
-   .. tab:: IR
-   
-      .. code-block:: c
-   
-         auto network  = core.ReadNetwork("model.xml");
-
-   .. tab:: ONNX
-      
-      .. code-block:: c
-         
-         auto network = core.ReadNetwork("model.onnx");
-
-      You can find more information about the ONNX format support in the document `ONNX format support in the OpenVINO™ <https://docs.openvino.ai/latest/openvino_docs_IE_DG_ONNX_Support.html>`_   
-   
-   .. tab:: nGraph
-      
-      .. code-block:: c
-         
-         std::shared_ptr<Function> createNetwork() {
-            // To construct a network, please follow 
-            // https://docs.openvino.ai/latest/openvino_docs_nGraph_DG_build_function.html
-         }
-         auto network = CNNNetwork(createNetwork());
+   - IR:
+        @snippet snippets/src/main.cpp part1_1
+   - ONNX:
+        @snippet snippets/src/main.cpp part1_2
+   - Paddle:
+        @snippet snippets/src/main.cpp part1_3
+   - OpenVINO Model:
```

Comment:
=> OpenVINO model in memory (can be read via Core::read_model or create manually in code, see build ngraph function sample) ?